Source/Scene/Cesium3DTile.js

```diff
@@ -220,13 +221,23 @@ function Cesium3DTile(tileset, baseResource, header, parent) {
       );
       contentHeaderUri = contentHeader.url;
     }
-    contentState = Cesium3DTileContentState.UNLOADED;
-    contentResource = baseResource.getDerivedResource({
-      url: contentHeaderUri,
-    });
-    serverKey = RequestScheduler.getServerKey(
-      contentResource.getUrlComponent()
-    );
+
+    if (contentHeaderUri === "") {
+      console.log(
```

Comment:
should this use `oneTimeWarning` instead? there's an eslint error because the import is unused.